components/badge/badge.twig

```diff
@@ -2,7 +2,7 @@
   <div class="umami-badge__container">
     {% if icon %}
       <div class="umami-badge__icon">
-        {{ source(componentMetadata.path ~ '/icons/' ~ icon|default('knife') ~ '.svg') }}
+        {{ source('@umami/badge/icons/' ~ icon|default('knife') ~ '.svg') }}
```

Comment:
i would like to keep at least example of 
`componentMetadata` in repo. 
probably we can create new component for that 
`@umami/badge/icons/` namespace?